pkg/apis/eventing/v1/broker_defaults_test.go

```diff
@@ -138,6 +167,8 @@ var (
 	}
 )
 
+/// FIXME: add the broker classes consideration into the test cases in this file
```

Comment:
@Leo6Leo are you planning on adding this in this PR, or will it be a follow up? I think this may clarify my earlier comment about using the broker class when setting the default

Reply:
Maybe add it in a separate PR? otherwise this PR will become too large.

Reply:
Sure, could you capture that in an issue? I don't want to lose it, as currently we aren't actually testing the behaviour added by this PR

Reply:
The issue has been created https://github.com/knative/eventing/issues/7660